Makefile

```diff
@@ -122,25 +125,25 @@ GOCOV ?= "-covermode=atomic -coverprofile REPLACE_FILE"
 GIT_COMMIT_ID ?= $(shell git rev-parse --short HEAD)
 
 OPERATOR_VERSION ?= 0.3.0
-OPERATOR_GROUP ?= ${GO_PACKAGE_ORG_NAME}
-OPERATOR_IMAGE ?= quay.io/${OPERATOR_GROUP}/${GO_PACKAGE_REPO_NAME}
-OPERATOR_IMAGE_REL ?= quay.io/${OPERATOR_GROUP}/app-binding-operator
+OPERATOR_REPO_REF ?= quay.io/redhat-developer/servicebinding-operator
```

Comment:
have we joined the service and binding as `servicebinding` on purpose?

Reply:
This name is temporary until we decomission old `appregistry`(used by obsolete operator sources) that occupies (internally) `quay.io/redhat-developer/service-binding-operator` image.

Reply:
Once the above `appregistry` is removed, we switch to the `quay.io/redhat-developer/service-binding-operator`